packages/components/src/components/segmented-control/segmented-control/segmented-control.tsx

```diff
@@ -67,6 +68,7 @@ export class SegmentedControl {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
+    warnIfBackgroundColorIsUsed(this.host, this.backgroundColor);
```

Comment:
Can we use the `warnIfDeprecatedPropValueIsUsed()` util here?

Reply:
@denyo true, I will fix it

Reply:
@denyo I couldn't use `warnIfDeprecatedPropValueIsUsed()` in this case, because in this case the property "background-color" is deprecated completely, not only some particular value. But I created another general function for it.